London | ITP-Sep-2025 | Mohammad_Jafarzadeh | Sprint 1 | Wireframe

Ashjz opened the pull request:
<!--

You must title your PR like this:

Region | Cohort | FirstName LastName | Sprint | Assignment Title

For example,

London | 25-ITP-May | Carol Owen | Sprint 1 | Alarm Clock

Fill in the template below - remove any sections that don't apply.

Complete the self checklist - replace each empty box in the checklist [ ] with a [x].

Add the label "Needs Review" and you will get review.

Respond to volunteer reviews until the volunteer marks it as "Complete".

-->

## Learners, PR Template

Self checklist

- [x] I have titled my PR with Region | Cohort | FirstName LastName | Sprint | Assignment Title
- [x] My changes meet the requirements of the task
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)

## Changelist

same page on May cohort

## Questions

no

jenny-alexander reviewed (commented):
Good job on working through the requirements! While the CSS and HTML file are done well, I would spend some time reviewing the layout requirements defined in the wireframe.
 
There are a few changes needed:
- Can you review the wireframe more carefully? Notice how the article title, summary and button are aligned to the left. Can you modify your application to look like this?

<img width="322" height="233" alt="Image" src="https://github.com/user-attachments/assets/88da33e0-6180-469d-a9b1-757185c4dce0" />

- The image for each article should span the width of the container it's contained within. Right now, your images are center-aligned.
- The font for 'README' is slightly larger than the page title font. The page title font should be the largest on the page. 
- Can you shorten the Page summary ('Take a look at the answers to some questions...')? Right now, it's a little bit too long and doesn't match the wireframe requirements.

Ashjz commented:
> @Ashjz
> 
> Why did you close this [reviewed PR](https://github.com/CodeYourFuture/Module-Onboarding/pull/739)? Different reviewers, having different specialties, may focus on different things which you could improve. Can you address @LonMcGregor comments in this PR (since you have already closed the other PR) in addition to addressing the comments given by @jenny-alexander?

because it was the wrong pr actually

cjyuan commented:
> > @Ashjz
> > Why did you close this [reviewed PR](https://github.com/CodeYourFuture/Module-Onboarding/pull/739)? Different reviewers, having different specialties, may focus on different things which you could improve. Can you address @LonMcGregor comments in this PR (since you have already closed the other PR) in addition to addressing the comments given by @jenny-alexander?
> 
> because it was the wrong pr actually

What do you mean by "wrong pr". That PR and this PR shared the same branch.
